sdk/servicebus/test-resources.json

```diff
@@ -141,9 +145,9 @@
     {
       "type": "Microsoft.ServiceBus/namespaces/topics/subscriptions",
       "apiVersion": "2017-04-01",
-      "name": "[concat(variables('namespaceName'), '/', variables('topicName'), '/', variables('subscriptionName'), '-', copyIndex())]",
+      "name": "[concat(variables('namespaceName'), '/', variables('topicName'), '-', copyIndex(), '/', variables('subscriptionName'), '-0')]",
```

Comment:
Do we need the copyIndex() if we are just going to create one subscription and session subscription for each.

Reply:
We still need copyIndex() for the topic name. But I will remove `-0` for subscription name.